docs/table/mixin_columns.rst

```diff
@@ -153,6 +153,40 @@ You can conveniently convert |Table| to |QTable| and vice-versa::
 
 .. EXAMPLE END
 
+.. EXAMPLE START: Combining Units and Quantities
+
+In the following example, a |QTable| is used to store data as :ref:`user-defined
+units <astropy:defining_units>` in a FITS file.
+
+  >>> import numpy as np
+  >>> lps = u.def_unit('Lps', u.L / u.s)
+  >>> qt3 = QTable()
+  >>> qt3['speeds'] = np.arange(5) * lps
+  >>> qt3.write('Lps_output.fits', overwrite=True, format='fits')
+
+In order to parse and read the file contents, it is necessary to enable the
+new defined unit by calling :func:`~astropy.units.add_enabled_units`, otherwise
+the unit will fail to parse::
+
+  >>> QTable.read('Lps_output.fits', format='fits')
```

Comment:
The `+SHOW_WARNINGS` directive should make the warning appear in the doctest., and saving the table in a variable is probably more representative of how `QTable.read()` is typically used.
```suggestion
  >>> qt4 = QTable.read('Lps_output.fits', format='fits')  # doctest: +SHOW_WARNINGS
```